ansible_collections/arista/avd/plugins/plugin_utils/schema/avdtodocumentationschemaconverter.py

```diff
@@ -0,0 +1,255 @@
+        output = []
+        schema_items = schema.get('items', {})
+        schema_items_type = schema_items.get('type')
+        if schema_items_type == "dict":
+            schema_keys = schema_items.get('keys', [])
+            first = True
+            for key, value in schema_keys.items():
+                rows = self.build_table_row(
+                    var_name=key,
+                    schema=value,
+                    indentation=f"{indentation}    ",
+                    parent_schema=schema,
+                    first_list_item_key=first
+                )
+                output.extend(rows)
+                first = False
+        else:
+            output = self.build_table_row(
+                var_name=f"<{schema_items_type}>",
+                schema=schema_items,
+                indentation=f"{indentation}    ",
+                parent_schema=schema,
+                first_list_item_key=True
+            )
+        return output
+
+    def required(self, schema: dict, var_name: str, parent_schema: dict):
+        output = None
+        if schema.get('required'):
+            output = "Required"
+            if parent_schema and parent_schema.get('primary_key') == var_name:
+                output = f"{output}, Unique"
+        return output
+
+    def default(self, schema: dict):
+        return schema.get('default')
+
+    def restrictions(self, schema: dict):
+        restrictions = []
+        if schema.get('min'):
+            restrictions.append(f"Min: {schema['min']}")
+        if schema.get('max'):
+            restrictions.append(f"Max: {schema['max']}")
+        if schema.get('min_length'):
+            restrictions.append(f"Min Length: {schema['min_length']}")
+        if schema.get('max_length'):
+            restrictions.append(f"Max Length: {schema['max_length']}")
+        if schema.get('format'):
+            restrictions.append(f"Format: {schema['format']}")
+        if schema.get('dynamic_valid_values'):
+            schema.setdefault("valid_values", [])
+            valid_value = f"<value(s) of {schema['dynamic_valid_values']}>"
+            schema["valid_values"].append(valid_value)
+        if schema.get('valid_values'):
+            restrictions.append("Valid Values:")
+            for valid_value in schema['valid_values']:
+                restrictions.append(f"- {valid_value}")
+        if schema.get('pattern'):
+            restrictions.append(f"Pattern: {schema['pattern']}")
+
+        if restrictions:
+            return "<br>".join(restrictions)
+        return None
+
+    def description(self, schema: dict, indentation: str):
+        descriptions = []
+        if schema.get("display_name"):
+            descriptions.append(schema["display_name"])
+        if schema.get("description") and indentation:
+            descriptions.append(str(schema["description"]).replace("\n", "<br>"))
+
+        if descriptions:
+            return "<br>".join(descriptions)
+        return None
```

Comment:
Depending on how `None` is used later we could simply write:
```suggestion
        return "<br>".join(descriptions)
```

That would return an empty string if the list is empty

Reply:
It is used in the documentation jinja template to detect if the description field is set. Using arista.avd.defined. So an empty string will not work.